sqlglot/dialects/spark.py

```diff
@@ -170,6 +171,9 @@ class Generator(Spark2.Generator):
             exp.TryCast: lambda self, e: (
                 self.trycast_sql(e) if e.args.get("safe") else self.cast_sql(e)
             ),
+            exp.Levenshtein: unsupported_args("ins_cost", "del_cost", "sub_cost", "max_dist")(
+                rename_func("LEVENSHTEIN")
+            ),
```

Comment:
The Spark dialect inherits from Hive so we don't have to redefine it's builders & constructors unless they differ
```suggestion
```